snapshot/policy/retention_policy.go

```diff
@@ -160,6 +168,14 @@ func (r *RetentionPolicy) getRetentionReasons(i int, s *snapshot.Manifest, cutof
 			continue
 		}
 
+		if *r.KeepMinDays != *newOptionalInt(0) {
```

Comment:
Nit: is it necessary to compare to zero here? What happens if we remove lines `154-155` and do this:
```suggestion
		if r.KeepMinDays != nil {
```